### PR TITLE
Fixed multiple events per change, added dynamic positioning of watchers

### PR DESCRIPTION
**Pull request by SindriTh**

### CODE REVIEW

File: cockpit/IPC/src/views/Connections.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ const Connections:React.FC<DarkModeType> = ({ isDark }) => {
   return (
     <>
       <Title headingLevel="h1" size="2xl" style={{ color: isDark ? '#EEE' : '#111' }}>
-        IPC - Time For Change
+        Connections
```
Comment (omarhogni) [inline cockpit/IPC/src/views/Connections.tsx:32]: Erum við alveg á þessu?

File: cockpit/IPC/src/views/Configurator.tsx [hunk 2]
```diff
@@ -270,7 +270,7 @@ const Configurator: React.FC<DarkModeType> = ({ isDark }) => {
                 size="2xl"
                 style={{ marginBottom: '2rem', color: isDark ? '#EEE' : '#111' }}
               >
-                Configurator - Time For Change
+                Configurator
```
Comment (omarhogni) [inline cockpit/IPC/src/views/Configurator.tsx:273]: Same

**Review by omarhogni — APPROVED** [review]

File: cockpit/IPC/src/views/Connections.tsx (hunk 1 above)
Comment (SindriTh) [reply cockpit/IPC/src/views/Connections.tsx:32]: Felt unnecessarily long, and caused problems on mobile.
Could make font smaller, but It looks odd.

Suggestions?

**Comment by jbbjarnason** [comment]
Defer naming problem, ideally creating subsection for TFC would be perfect. Merging.

**Pull request merged**